code/modules/paperwork/paper.dm

```diff
@@ -474,7 +474,12 @@
 		if(clipboard.pen)
 			holding = clipboard.pen
 
-	data["held_item_details"] = holding?.get_writing_implement_details()
+	//NSV13 - Old part here runtimes.
+	if(holding)
+		data["held_item_details"] = holding.get_writing_implement_details()
+	else
+		data["held_item_details"] = null //Overriding old data, otherwise data would not get changed.
+	//NSV13 end.
```

Comment:
Could make this a ternary like:
```suggestion
	data["held_item_details"] = holding ? holding.get_writing_implement_details() : null //NSV13 Overriding old data, otherwise data would not get changed.
```
But yours is also valid

Reply:
I think I prefer this as an elif to emphasize the "passing null does something specific" for whoever has to do a beebase that touches this.